src/java.security.jgss/share/classes/sun/security/krb5/KrbTgsReq.java

```diff
@@ -318,7 +318,7 @@ private TGSReq createRequest(
         byte[] temp = reqBody.asn1Encode(Krb5.KRB_TGS_REQ);
         // if the checksum type is one of the keyed checksum types,
         // use session key.
```

Comment:
nit: does this comment need updating?

Reply:
Will remove it. I think there is no need to explain why -1 is used.

Reply:
Sure